### DISCUSSION TITLE
Parallel installation of same JDK version for multiple architectures

### DISCUSSION THREAD

**Discussion by kwin** [adoptium/adoptium-support#615] [Q&A] [answered]
The Apple Silicon computers support two different architectures (x86-64 and aarch64) through Rosetta 2. For some edge cases (calling binary code not yet supporting aarch64) I still need to rely on the slower x86-64 architecture. Unfortunately both use the same destination directory so a parallel installation is impossible. How can I achieve having both installed in parallel?

**Comment by karianna** [comment, marked as answer]
Are you installing via brew?

**Reply by kwin** [reply]
Manually with the installer from the DMG file.

**Reply by karianna** [reply]
OK, taking this to the PMC for discussion

**Reply by karianna** [reply]
@kwin We discussed this at the PMC and it's a niche use case so our advice here is to use the tar.gz packages and install them in seperate directories that way as a workaround but we won't be changing the installers.

**Reply by kwin** [reply]
Understood, makes sense. Thanks anyways.
